gr-filter/lib/filterbank_vcvcf_impl.cc

```diff
@@ -63,7 +64,8 @@ int filterbank_vcvcf_impl::general_work(int noutput_items,
         return 0; // history requirements may have changed.
     }
 
-    std::vector<gr_complex> working(noutput_items + d_ntaps);
+    gr_complex* working = (gr_complex*)volk_malloc(
```

Comment:
Instead of `volk_malloc` you can use `volk::vector<gr_complex>` that uses `volk_malloc`. You can still pass pointers around via `vec.data()` but you don't need manual memory management anymore.

Reply:
Oh, nice! I'll make that change.